cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -582,10 +582,19 @@ public void onClose(KubernetesClientException e) {
                                     //       an ongoing rebalance. We may need to add an Error state to the state machine.
                                     apiClient.getUserTaskStatus(host, CruiseControl.REST_API_PORT, sessionId).setHandler(userTaskResult -> {
                                         if (userTaskResult.succeeded()) {
-                                            CruiseControlResponse response = userTaskResult.result();
+                                            CruiseControlUserTaskResponse response = userTaskResult.result();
                                             JsonObject taskStatusJson = response.getJson();
-                                            String taskStatusStr = taskStatusJson.getString("Status");
-                                            CruiseControlUserTaskStatus taskStatus = CruiseControlUserTaskStatus.lookup(taskStatusStr);
+                                            CruiseControlUserTaskStatus taskStatus;
+                                            // Due to a bug in the CC rest API, if we ask for the status of a task that has COMPLETED_WITH_ERROR
```

Comment:
is this bug tracked or we can open an issue on CC github and putting a reference here?

Reply:
Ok, I can do that